routes/likedVideos.router.js

```diff
@@ -0,0 +1,114 @@
+const express = require('express');
+const router = express.Router();
+const { LikedVideo } = require('../models/likedVideo.model');
+const { User } = require('../models/user.model');
+
+router.get('/', async (req, res) => {
+    try {
+        const likedVideos = await LikedVideo.find({}).populate('videos');
+        res.json({ success: true, likedVideos });
+    } catch (error) {
+        console.log(error);
+        res.status(404).json({
+            success: false,
+            error: 'could not get liked videos',
+        });
+    }
+});
+
+//function to check if the userId is corret and then adding it to req
+router.param('userId', async (req, res, next, userId) => {
+    try {
+        const user = await User.findById(userId);
+        // console.log(user);
```

Comment:
Omit commented code. It's better to get them removed.

Reply:
Can use webpack to bundle the backend code too :P